samples/azure-service-bus-netstandard/lock-renewal/ASBS_2/LockRenewal/LockRenewalBehavior.cs

```diff
@@ -4,15 +4,13 @@
 using Azure.Messaging.ServiceBus;
 using NServiceBus.Logging;
 using NServiceBus.Pipeline;
-using NServiceBus.Transport;
 
 class LockRenewalBehavior : Behavior<ITransportReceiveContext>
 {
-    public LockRenewalBehavior(TimeSpan lockDuration, TimeSpan renewLockTokenIn, string queueName)
+    public LockRenewalBehavior(TimeSpan lockDuration, TimeSpan renewLockTokenIn)
```

Comment:
Lock duration is preset on the queue to 5 minutes. This gives users the mistaken impression that they get to set lock duration on a per-message basis. Can't we just change this to:
```suggestion
    public LockRenewalBehavior(TimeSpan renewAfter)
```

Where `renewAfter` must be less than 5 minutes?

Reply:
I personally like that approach, that would in theory allow a renewal to fail and be retried too.